nevergrad/functions/images/test_core.py

```diff
@@ -7,12 +7,64 @@
 import numpy as np
 import nevergrad as ng
 from . import core
+from torchvision.models import resnet50
+import torch.nn as nn
+import torch
+import torchvision.transforms as transforms
+import torchvision.datasets as datasets
+
+
+class Normalize(nn.Module):
+    def __init__(self, mean, std):
+        super(Normalize, self).__init__()
+        self.mean = torch.Tensor(mean)
+        self.std = torch.Tensor(std)
+
+    def forward(self, x):
+        return (x - self.mean.type_as(x)[None, :, None, None]) / self.std.type_as(x)[None, :, None, None]
+
+
+class Classifier(nn.Module):
+    def __init__(self):
+        super(Classifier, self).__init__()
+        self.norm = Normalize(mean=[0.485, 0.456, 0.406],
+                              std=[0.229, 0.224, 0.225])
+        self.model = resnet50(pretrained=True)
+
+    def forward(self, x):
+        return self.model(self.norm(x))
+
+
+def test_images_adversarial() -> None:
+    image_size = 224
+    classifier = Classifier()
+    classifier.cuda()
+    classifier.eval()
+    classifier = torch.nn.DataParallel(classifier, device_ids=range(torch.cuda.device_count()))
+    image_folder = datasets.ImageFolder("/datasets01_101/imagenet_full_size/061417/val",
```

Comment:
Hard path cannot be used. For tests you need some fake data

Reply:
For this to work everywhere, you might use "if".
E.g." if /datasets01_101 exists then use it, otherwise use xxxx1.jpg and xxx2.jpg (with these two images inside Nevergrad)"